packages/pds/src/api/com/atproto/sync/subscribeRepos.ts

```diff
@@ -66,13 +66,6 @@ export default function (server: Server, ctx: AppContext) {
           time: evt.time,
           ...evt.evt,
         }
-      } else if (evt.type === 'tombstone') {
-        yield {
-          $type: '#tombstone',
-          seq: evt.seq,
-          time: evt.time,
-          ...evt.evt,
-        }
```

Comment:
What's the intended interaction between old data in the sequencer table and this new logic.  Looks like this should just ignore data that has already been stored for the deprecated event types?

Reply:
yeah exactly, it'll just skip over it